scheduler/kubernetes/elasticstack-kibana-k8s.md

```diff
@@ -143,70 +151,76 @@ spec:
           name: transport
           protocol: TCP
         volumeMounts:
-        - name: storage
+        - name: px-storage
           mountPath: /data
-      volumes:
-          - emptyDir:
-              medium: ""
-            name: "storage"
+  volumeClaimTemplates:
+  - metadata:
+      name: px-storage
+      annotations:
+        volume.beta.kubernetes.io/storage-class: px-es-master-sc
+    spec:
+      accessModes: [ "ReadWriteOnce" ]
+      resources:
+        requests:
+          storage: 80Gi
```

Comment:
80Gi might be too high. Most people use these specs for a PoC and they may not have a big cluster.

Reply:
Volumes are thin provisioned, so it doesn't matter